ladybug/src/main/java/nl/nn/ibistesttool/IbisDebuggerAdvice.java

```diff
@@ -74,17 +76,20 @@
 public class IbisDebuggerAdvice implements ThreadLifeCycleEventListener<ThreadDebugInfo>, ApplicationListener<DebuggerStatusChangedEvent>, IXmlDebugger {
 	protected Logger log = LogUtil.getLogger(this);
 
+	private @Setter IbisDebugger ibisDebugger;
+	private @Setter IbisManager ibisManager;
+
 	// Contract for testtool state:
 	// - when the state changes a DebuggerStatusChangedEvent must be fired to notify others
 	// - to get notified of changes, components should listen to DebuggerStatusChangedEvents
 	// IbisDebuggerAdvice stores state in appconstants testtool.enabled for use by GUI
-	private IbisDebugger ibisDebugger;
 	private static boolean enabled=true;
 	
 	private AtomicInteger threadCounter = new AtomicInteger(0);
-	
-	public void setIbisDebugger(IbisDebugger ibisDebugger) {
-		this.ibisDebugger = ibisDebugger;
+
+	public void init() {
```

Comment:
Zou je niet beter de `InitializingBean` interface kunnen implementeren? Die heeft een `afterPropertiesSet()` methode die ook als een soort init of configure functioneert.

Reply:
Done